android/res/values/strings.xml

```diff
@@ -104,6 +104,8 @@
   <string name="preferences_disable_barcode_scene_mode_title">No barcode scene mode</string>
   <string name="preferences_disable_continuous_focus_summary">Use only standard focus mode</string>
   <string name="preferences_disable_continuous_focus_title">No continuous focus</string>
+  <string name="preferences_disable_history">Barcode history is not automatically saved</string>
```

Comment:
All the new strings. Hm, I remembered though that you have to have an API key for Google Translate. It only costs cents but probably not worth you bothering. Unless you add them by hand I can just run the tool later.